rotkehlchen/chain/ethereum/modules/compound/decoder.py

```diff
@@ -207,6 +216,78 @@ def _decode_borrow_and_repay(
 
         return DEFAULT_DECODING_OUTPUT
 
+    def _decode_liquidate(
+            self,
+            transaction_hash: 'EVMTxHash',
+            tx_log: 'EvmTxReceiptLog',
+            decoded_events: list['EvmEvent'],
+            all_logs: list['EvmTxReceiptLog'],
+    ) -> DecodingOutput:
+        """Decode a liquidation event happening over a tracked account"""
+        borrower = hex_or_bytes_to_address(tx_log.data[32:64])
+        if self.base.is_tracked(borrower) is False:
```

Comment:
what about the liquidator's side? What if he is using rotki?

Reply:
Added it to handle the normal path. For liquidations happening using MEV bots we have to make additional changes to our logic